contracts/ERC20Safe.sol

```diff
@@ -50,6 +50,7 @@ contract ERC20Safe is BridgeRole, Pausable {
     mapping(uint256 => Deposit[]) public batchDeposits;
 
     event ERC20Deposit(uint112 depositNonce, uint112 batchId);
+    event ERC20SCDeposit(uint112 depositNonce, uint112 batchId, string callData);
```

Comment:
have you forgotten to specify that the depositNonce is indexed? The filtering does not work.

Reply:
and also they are in reversed order.
Original:
```event ERC20SCDeposit(uint64 indexed batchNonce, uint64 depositNonce, string callData);```
versus
```event ERC20SCDeposit(uint112 indexed depositNonce, uint112 batchId, string callData);```